.github/contributing.md

```diff
@@ -16,25 +16,40 @@ This document outlines the different ways you can contribute to the UNiDAYS repo
 [How Can I Contribute?](#how-can-i-contribute)
   * [Reporting Bugs](#reporting-bugs)
   * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Labels For Issues](#labels-for-issues)
   * [Pull Requests](#pull-requests)
   * [Your First Code Contribution](#your-first-code-contribution)
 
-[Styleguides](#styleguides)
+[Style Guides](#style-guides)
   * [Git Commit Messages](#git-commit-messages)
+  * [Documentation Style Guide](#documentation-style-guide)
 
 ## How Can I Contribute?
 
-### Reporting Bugs :bug:
+### Reporting Bugs
 
-When issuing a bug report please include as many of details on [the bug report template](bug_report.md) as possible. If you find a Closed issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one. Please label the any bug report issues with `bug`.
+When issuing a bug report please include as many of details on [the bug report template](./ISSUE_TEMPLATE/bug_report.md) as possible. If you find a Closed issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one. Please label the any bug report issues with `bug`.
 
-### Suggesting Enhancements 🔎
+### Suggesting Enhancements
 
 Enhancements may include completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related suggestions.
 
-First of all please do a search on the issues already present to see if it's an enhancement that hasn't previously been suggested. If it has, maybe join the discussion on the pre-existing issue. This is to help reduce duplication of issues. If it hasn't been raised previously, please include as many details as you can using [the feature request template](feature_request.md). Label the raised feature request as `enhancement`
+First of all please do a search on the issues already present to see if it's an enhancement that hasn't previously been suggested. If it has, maybe join the discussion on the pre-existing issue. This is to help reduce duplication of issues. If it hasn't been raised previously, please include as many details as you can using [the feature request template](./ISSUE_TEMPLATE/feature_request.md)). Label the raised feature request as `enhancement`
 
-### Pull Requests :thought_balloon:
+### Labels For Issues
+
+| Label       | Purpose             |

```

Comment:
I feel like the justification on these tables need to just be left aligned. But that's a stylistic decision

Reply:
All the tables need looking at in this. They need some consistency.